CROSSSECTOR/DIH/DigitalInnovationHub/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Option 1) File name is wrong. then you should rename example-nomalized.json and remove @context 
option 2) You should change all the types to Property and keep the name.